### PR TITLE
feat(dmn): open dmn files inside of the idea in a custom dmn editor

### PR DESCRIPTION
**Pull request by rami-nk**
**Issue**
Dmn files could not be opened inside of a dmn editor inside of the IDE.

**Description**
This PR provides the functionality of opening dmn files inside of a dmn editor using dmn-js.

**Checklist**

* Code is easy to understand
* No obvious errors or bugs
* CI/CD Pipelines did run successfully
* Tests were implemented
* Documentation was created

### CODE REVIEW

File: apps/miranum-dmn-modeler-webview/src/main.ts [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+            const message =
+                error instanceof Error ? error.message : "Could not handle message";
+            postMessage(MessageType.ERROR, undefined, message);
+        }
+    });
+
+    waitForActiveViewToBeLoaded().then(() => {
+        const activeEditor = modeler.getActiveViewer();
+        const eventBus = activeEditor.get("eventBus");
+        eventBus.on("commandStack.changed", sendChanges);
```
Comment (peterhnm) [inline apps/miranum-dmn-modeler-webview/src/main.ts:165]: Strangely, this is not triggered when the user changes something in the decision table. Maybe there is some other event that enables this.

**Review by peterhnm — CHANGES_REQUESTED** [review]
👍🏻 Good work. 
There is a problem when the user changes something in the decision table. The event we are listening for is not triggered and the changes are not saved in the .dmn file.

File: apps/miranum-dmn-modeler-webview/src/main.ts (hunk 1 above)
Comment (rami-nk) [reply apps/miranum-dmn-modeler-webview/src/main.ts:165]: Ok, very good find! I will definitively have a look at it. Ah I maybe know where to look for it

Comment (peterhnm) [reply apps/miranum-dmn-modeler-webview/src/main.ts:165]: Maybe [this](https://forum.bpmn.io/t/what-is-the-event-for-model-change-in-dmn-modeler/5246) forum post and the link to the [CodeSandbox](https://codesandbox.io/s/dmn-js-elements-changed-5efs8?file=/index.html) helps.

**Review by peterhnm — APPROVED** [review]
👍🏻 Really good

**Pull request merged**